conpot/conpot.cfg

```diff
@@ -39,6 +39,14 @@ host = taxiitest.mitre.org
 port = 80
 inbox_path = /services/inbox/default/
 use_https = False
+
+[stix]
+port_path_list = [
+    "//conpot_template/protocols/modbus/@port",
```

Comment:
@johnnykv Why do you feel uncomfortable having them in config file ?
@glaslos @johnnykv  Is it better like this :

    port_path_modbus = //conpot_template/protocols/modbus/@port
    port_path_snmp = //conpot_template/protocols/snmp/@port
    port_path_http = //conpot_template/protocols/http/@port
    port_path_s7 = //conpot_template/protocols/s7comm/@port

in config.cfg ? What do you suggest ?